diff --git a/node_modules/@now/build-utils/fs/run-user-scripts.js b/node_modules/@now/build-utils/fs/run-user-scripts.js
index a71a564..14d5d93 100644
--- a/node_modules/@now/build-utils/fs/run-user-scripts.js
+++ b/node_modules/@now/build-utils/fs/run-user-scripts.js
@@ -73,7 +73,11 @@ async function installDependencies(destPath, args = []) {
     await spawnAsync('npm', ['install'].concat(commandArgs), destPath);
     await spawnAsync('npm', ['cache', 'clean', '--force'], destPath);
   } else {
-    await spawnAsync('yarn', ['--cwd', destPath].concat(commandArgs), destPath);
+    await spawnAsync('yarn', [
+      '--cwd', destPath,
+      '--cache-folder', '.yarn-cache',
+      '--silent'
+    ].concat(commandArgs), destPath);
     await spawnAsync('yarn', ['cache', 'clean'], destPath);
   }
 }
